<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgBackgrounds</class>
 <widget class="QWidget" name="WdgBackgrounds">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Size</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QListWidget" name="lstBackgrounds">
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="bnAdd">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="bnRemove">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="bnReset">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lstBackgrounds</tabstop>
  <tabstop>bnAdd</tabstop>
  <tabstop>bnRemove</tabstop>
  <tabstop>bnReset</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
